docs/001_develop/03_client-capabilities/010_filters/foundation-filters.mdx

```diff
@@ -0,0 +1,437 @@
+* **timeWindow**<!-- TODO: [timeWindow](./docs/api/foundation-filters.timewindowfilter.md) -->: This filter is used to check whether the current date and time come in between the start and end period specified in the parameters.The filter returns true if the current date and time falls between the range, thus making it useful for time-sensitive conditions.
+* **urlTargetting**<!-- TODO: [urlTargeting](./docs/api/foundation-filters.urltargetingfilter.md) -->: It is used to filter content based on URL parts like scheme, path, host, or pattern matching. It allows to target specific URLs based on particular criteria, which can be useful for enabling or restricting features.
+* **userAgent**<!-- TODO: [userAgent](./docs/api/foundation-filters.useragentfilter.md) -->: It filters content based on user's browser, OS, devices or specified user agent string. It allows customized expereince according to specific environments, such as targetting mobile devices audience or restricting access on a particular browser.
+* **userTargeting**<!-- TODO: [userTargeting](./docs/api/foundation-filters.usertargetingfilter.md) -->: This filters content on basis of user related attributes like usernames, permissions and profiles. It aids in showing personalized content or restricting access to specific content depending on the soecified user conditions.
+
+
+## Usage
+
+Each filter is provided in two flavours:
+1. Functions
+2. Dependency Injection(DI)
+
+These flavours are explained in detail below with help of code examples
+<FiltersDemo />
+### As Function
+
+    ```typescript
+    import { timeWindowFilter } from '@genesislcap/foundation-filters';
+
+    @customElement({
+      name: 'my-element',
+      styles: css`
+        .container {
+          display: flex;
+          flex-direction: column;
+          gap: calc(var(--design-unit) * 3px);
+          padding: calc(var(--design-unit) * 4px);
+        }
+      `,
+      template: html`
+        <div class="container">
+          <rapid-text-field type="datetime-local" :value=${sync((x) => x.startDate)}>
+            Start Period
+          </rapid-text-field>
+          <rapid-text-field type="datetime-local" :value=${sync((x) => x.endDate)}>
+            End Period
+          </rapid-text-field>
+          <rapid-button @click=${(x) => x.checkWindow()}>Check</rapid-button>
+          <span>${(x) => x.resultString}</span>
+        </div>
+      `,
+    })
+    export class MyElement extends GenesisElement {
+      @observable startDate: string;
+      @observable endDate: string;
```

Comment:
Can you set some default strings here so we can test the button straight away, and the user knows what format to write the dates?
```suggestion
      @observable startDate: string;
      @observable endDate: string;
```